go/teams/service_helper.go

```diff
@@ -121,18 +121,25 @@ func userVersionsToDetails(ctx context.Context, g *libkb.GlobalContext, uvs []ke
 	for i, uv := range uvs {
 		pkg := packages[i]
 		active := true
+		isDeleted := false
 		var fullName keybase1.FullName
 		if pkg.FullName != nil {
 			if pkg.FullName.EldestSeqno != uv.EldestSeqno {
 				active = false
+				// Check if this user was deleted or just reset
+				_, err := loadUserVersionByUsername(ctx, g, pkg.NormalizedUsername.String())
```

Comment:
I'm not sure if it's good, as this would trigger reload of that user (`forceRepoll: true` in `loadUserVersionByUIDCheckUsername`) for every member of the team, every time team is loaded until that user is removed from the team which might be never.